src/ionOverlayMethodCreateShouldUseAwaitRule.ts

```diff
@@ -0,0 +1,48 @@
+import * as Lint from 'tslint';
+import * as tsutils from 'tsutils';
+import * as ts from 'typescript';
+import { isValidForRule } from './helpers/parametersRenamed';
+export const ruleName = 'ion-overlay-method-create-should-use-await';
+export const ruleMessage = 'You must await the create method';
```

Comment:
Can this be more informative? 

e.g. "The create method of overlay controllers is now promise-based and it appears the returned promise is unhandled."